lib/api/handler/pokemon_api.dart

```diff
@@ -9,5 +13,21 @@ class PokemonApi {
   final ApiClient apiClient;
   final Uri baseUri;
 
-//TODO: to be continued...
-}
+  Future<List<Pokemon>> getPokemonList({
+    required String offset,
+    required String limit,
+  }) async {
+    final queryParameters = <String, dynamic>{};
+
+    queryParameters[queryParamLimit] = limit;
+    queryParameters[queryParamOffset] = offset;
+
+    final uri = baseUri.replace(
+        queryParameters: queryParameters, path: '${baseUri.path}$endPointPokemon');

```

Comment:
is the white space here needed?

Reply:
no need for `endPointPokemon`, I'm fine having the hard-coded interpolated string here so I can immediately see the endpoint value 🙂